ScnSoft-Education/LeetCodeTraining.Tests/MergeTwoSortedListsTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using LeetCodeTraining.MergeTwoSortedLists;
+
+namespace LeetCodeTraining.Tests
+{
+    public class MergeTwoSortedListsSolutionTests
+    {
+        public static ListNode InitializeNode(int[] numbers)
```

Comment:
Can you move this code and set up to ListNode constructor for initialization?

`public ListNode(int[] numbers)`